js/quotechew.js

```diff
@@ -113,8 +130,42 @@ var RE_SECTION_DELIM = /^[_-]{6,}$/;
 
 var RE_LIST_BOILER = /mailing list$/;
 
+/**
+ * Non-localized hard-coded heuristic to help us detect whether a line is the
+ * lead-in to a quote block.  This is only checked against the last content line
+ * preceding a quote block.  This only results in styling guidance, so if it's
+ * too aggresive or insufficiently aggressive, the worst that happens is the
+ * line will be dark gray instead of black.  (For now!  Moohoohahahahahaha.)
+ */
 var RE_WROTE_LINE = /wrote/;
 
+/**
```

Comment:
The locale-specific stuff here is a bit unfortunate -- the uses of these will only help in the English case, so seems of shallower benefit. However, I can appreciate the forces involved with communicating the locale strings. 

I am encouraged by the app_logic approach in convoy, where we could do the same with localization strings for this, and also for a replacement for mailapi's useLocalizedStrings: allow the front end to provide a module with localizations inside of it, then it is a simple module dependency relationship here.

For now though, maybe just a note that we want to improve this for other locales, but want to wait on the larger convoy efforts that may allow communicating the locale strings efficiently. Otherwise, this seems to stick out a bit as a bit suspicious.